tests/ruby/RuboCopBearTest.py

```diff
@@ -40,3 +41,33 @@ def bad_name
                                 valid_files=(camelCaseFile,),
                                 invalid_files=(snake_case_file,),
                                 settings={"rubocop_config": rubocop_config})
+
+# Testing settings
+good_file = """
+def goodindent
+ # 1 space indent
+end
+""".splitlines(keepends=True)
+
+bad_file = """
+def badindent
+  # 2 spaces indent
+end
+""".splitlines(keepends=False)
+
+RuboCopBearSettingsTest = verify_local_bear(
+                              RuboCopBear,
+                              valid_files=(good_file,),
+                              invalid_files=(bad_file,),
+                              settings={"indentation_width": 1})
+bad_file = """
+'using single-quotes'
+""".splitlines(keepends=True)
+good_file = """
+"using double-quotes"
+""".splitlines(keepends=True)
+RuboCopBearSettingsTest = verify_local_bear(
+                              RuboCopBear,
+                              valid_files=(good_file,),
+                              invalid_files=(bad_file,),
+                              settings={"string_literals": "double_quotes"})
```

Comment:
I tend to disagree with @sils1297 
Its better to have more sample files each with only 1 possible Result. When using 1 big test file, things get messy and it is not a "unit"test anymore.

Possibly you could simplify your code for easier reading. Like `bad_file = ("'using single-quotes'",)` instead of the whole `"""` thing